lib/ask/project.ex

```diff
@@ -25,7 +30,17 @@ defmodule Ask.Project do
   """
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:name, :salt, :colour_scheme, :archived])
+    |> cast(params, [
+      :name,
+      :salt,
+      :colour_scheme,
+      :timezone,
+      :archived,
+      :initial_success_rate,
+      :eligibility_rate,
+      :response_rate,
+      :valid_respondent_rate
+    ])
     |> validate_colour_scheme
   end
```

Comment:
We should validate the rates (float between 0.0 and 1.0), and maybe the timezone (it should exist)?